vite.config.ts

```diff
@@ -1,5 +1,8 @@
 import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vite';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
Why are we using dotenv here?

Reply:
I was testing and accidentily left this in.